snapshots/readme.md

```diff
@@ -0,0 +1 @@
+This is a directory of directories, where each directory correspond to a set of similar openFoam cases, i.e. snapshots. 
```

Comment:
I suggest this (using markdown, which you must learn soon):

Under the `snapshots` folders, OpenFOAM cases data (and sometimes their generated solution snapshots) can be found.

The contents in each subfolders is as follows:
* [td_heat_conduction](./td_heat_conduction) contains time-dependent OF cases for heat conduction problems


You may want to re-assess if `snapshots` is the correct folder name. Maybe `OF_cases` is now a better name ....